install-containerd-helpers

```diff
@@ -27,7 +27,12 @@ function install_rpm_containerd() {
 	if command -v dnf5; then
 		dnf --version
 
-		dnf config-manager addrepo --save-filename=docker-ce.repo --from-repofile="${REPO_URL}"
+		# FIXME(thaJeztah); strip empty lines as workaround for https://github.com/rpm-software-management/dnf5/issues/1603
+		TMP_REPO_FILE="$(mktemp --dry-run)"
+		curl -fsSL "${REPO_URL}" | tr -s '\n' > "${TMP_REPO_FILE}"
```

Comment:
`tr -s '\n'`  only removes empty lines. Comment lines must also be removed. May be `grep '^[^#]'`.

Reply:
Thanks! Yes, I was considering comment-lines as well, but our `.repo` files don't have them, so I ultimately only accounted for the empty lines.


It should (hopefully 🤞) just be temporary, and we can revert the commit once fixed.


FWIW: I lifted those commits to a separate PR (mostly for visibility); and we just merged them; https://github.com/docker/docker-ce-packaging/pull/1059